docs/howto/garbage-collection/standalone-gc.md

```diff
@@ -0,0 +1,189 @@
+   More about that in the [Output](./standalone-gc.md#output) section.
+
+### Lab tests
+
+<TODO: update with final results once ready>
+
+Repository spec:
+
+- 100k objects
+- < 200 commits
+- 1 branch
+
+Machine spec:
+- 4GiB RAM
+- 8 CPUs
+
+In this setup, we measured:
+
+- Time: ~1 minute
+- Disk space: 120MiB
+
+## Installation
+
+### Step 1: Obtain Dockerhub token
+As an enterprise customer, you should already have a dockerhub token for the `externallakefs` user.
+If not, contact us at ___ (TODO: add mail/whaterver).
+
+### Step 2: Login to Dockerhub with this token
+```bash
+docker login -u <your token>
+```
+
+### Step 3: Download the docker image
+Download the image from the [lakefs-sgc](https://hub.docker.com/repository/docker/treeverse/lakefs-sgc/general) repository:
+```bash
+docker pull treeverse/lakefs-sgc:tagname
+```
+
+## Usage
+
+### AWS Credentials
+Currently, `lakefs-sgc` does not provide an option to explicitly set AWS credentials. It relies on the hosting machine
+to be set up correctly, and reads the AWS credentials from the machine.
+
+This means, you should set up your machine however AWS expects you to set it. \
+For example, by following their guide on [configuring the AWS CLI](https://docs.aws.amazon.com/cli/v1/userguide/cli-chap-configure.html).
+
+### Configuration
+The following configuration keys are available:
+
+| Key                        | Description                                                                                        | Default value                  | Possible values                                         |
+|----------------------------|----------------------------------------------------------------------------------------------------|--------------------------------|---------------------------------------------------------|
+| `logging.format`           | Logs output format                                                                                 | "text"                         | "text","json"                                           |
+| `logging.level`            | Logs level                                                                                         | "info"                         | "error","warn",info","debug","trace"                    |
+| `logging.output`           | Where to output the logs to                                                                        | "-"                            | "-" (stdout), "=" (stderr), or any string for file path |
+| `logging.file_max_size_mb` | Max file size for logs output (relevant only if `logging.output` is set to a file path)            | 102400 (100MiB)                | number                                                  |
+| `logging.files_keep`       | Number of files to keep for logs rotation (relevant only if `logging.output` is set to a file path | 100                            | number                                                  |
```

Comment:
I would remove these 2 options. More is less, we're not expecting significant amount of logs from this GC, we can always add these later

Reply:
Done